cmd/kyma/apply/function/function.go

```diff
@@ -103,32 +105,45 @@ func (c *command) Run() error {
 		return err
 	}
 
-	subscriptions, err := resources.NewSubscriptions(configuration)
-	if err != nil {
-		step.Failure()
-		return err
-	}
+	operators := []operator.Operator{}
 
-	kymaAddress, err := c.kymaHostAddress()
-	if err != nil {
-		step.LogErrorf("%s\n%s", err, "Check if your cluster is available and has Kyma installed.")
+	if configuration.Subscriptions != nil {
```

Comment:
The definition of the Subscriptions field looks like this:

```go
Subscriptions []Subscription    `yaml:"subscriptions,omitempty"`
```

It does mean that in this case, we should react on non-zero value as well ( not nil but empty map ). Example here:
https://go.dev/play/p/b44TCFzbzNw

```suggestion
	if len(configuration.Subscriptions) != 0 {
```